.github/PULL_REQUEST_TEMPLATE/new_ea_pr_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+`<description>`
+
+## Steps to Test
+
+1. `yarn && yarn setup`
+2. `yarn test packages/<composites|sources|targets>/<adapter-name>/test`
+3. ...
+
+## New Adapter Checklist
+
+- [ ] Adapter was created by following [Creating a New Adapter](../../CONTRIBUTING.md#creating-a-new-adapter)
+- [ ] Endpoints are built following the style of the [example adapters](../../packages/examples).
+- [ ] Unit and integration tests cover all helper methods and endpoints.
+- [ ] Input/outputs are built to spec.
+- [ ] Code is documented with `const description = "..."` in each endpoint, and `description: "..."` in `schemas/env.json`.
+- [ ] Input parameters are defined in `const inputParameters = {...}` and `export type TInputParameters = {...}` in each endpoint file.
+- [ ] API credentials (ex. `API_KEY` and `API_SECRET`) are found in `schemas/env.json`.
```

Comment:
These are not always required. I'd say something like "if required"

Reply:
I'd like to use the terminology "if applicable" or similar, since we should have them document the creds even if they're not required.